fabric-biomes-v1/src/main/java/net/fabricmc/fabric/impl/biomes/PredicatedBiomeEntry.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.biomes;
+
+import java.util.function.BiPredicate;
+
+import net.minecraft.world.biome.Biome;
+import net.minecraft.world.biome.layer.LayerRandomnessSource;
+
+class PredicatedBiomeEntry {
+	private final Biome biome;
+	private final BiPredicate<Biome, LayerRandomnessSource> predicate;
+	
+	PredicatedBiomeEntry(Biome biome, BiPredicate<Biome, LayerRandomnessSource> predicate) {
+		this.biome = biome;
+		this.predicate = predicate;
+	}
+	
+	Biome getBiome() {
```

Comment:
Call it getResultBiome or add comments to make this less confusing? I am super confused on my first look here.

Reply:
I'm calling it getBiome because the other biome entry class, WeightedBiomeEntry, calls it getBiome.

Reply:
Sure, this name is not that confusing in IDE or to those who actually use the code, and given this is a package-private impl detail this is totally fine!